.gitignore

```diff
@@ -1,3 +1,5 @@
 /composer.lock
 /phpunit.xml
 /vendor
+.idea
```

Comment:
This should be in your global git ignore rules, then you don't need to add it to every project you work on